scripts/e2e.sh

```diff
@@ -26,7 +26,7 @@ PROMETHEUS_VERSION=2.0.0
 mkdir -p $KUBE_STATE_METRICS_LOG_DIR
 
 # setup a Kubernetes cluster
-curl -sLo minikube https://storage.googleapis.com/minikube/releases/v0.25.2/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
+curl -sLo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
```

Comment:
I'd prefer to still explicitly bump minikube, there have been numerous occasions where the latest releases were broken to a point where it was just not usable at all.

Reply:
Agreed for now since kube-state-metrics do not need any new feature that latest minikube supports. Let's first pin at `v0.25.2` and seem whether Travis CI ubuntu image will support systemd alongside.